tdrs-backend/tdpservice/settings/common.py

```diff
@@ -435,23 +435,23 @@ class Common(Configuration):
     CELERY_BEAT_SCHEDULE = {
         'name': {
             'task': 'tdpservice.scheduling.tasks.postgres_backup',
-            'schedule': crontab(minute='*', hour='4'), # Runs at midnight EST
+            'schedule': crontab(minute='0', hour='5'), # Runs at midnight EST
```

Comment:
@elipe17 looks like we are also changing the backup schedule to run once at at 1 am ET? Is this correct? good catch! How frequently does this run?

Reply:
Updated it back to midnight. And previously that job was scheduled to run once per day and every minute from 12:00am - 12:59am EST which is obscene for a DB backup haha. But now it will run once per day at 12:00am EST

Reply:
Oh and Connor deserves the credit for catching this one!